src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -351,18 +350,6 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
   const isCreateAlertAvailable =
     artwork.isEligibleToCreateAlert && artwork.isSold
 
-  const AlertSwitch: FC = () => {
```

Comment:
I wonder how that gets passed through review, it shouldn't

Reply:
Probably just hard to notice with whitespace.

Reply:
Oh! https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

Reply:
I take [the blame](https://github.com/artsy/force/pull/13092/files#diff-022388e824526755e0276e0538a702780bf856cac90d5ca3c7d8e5dcc65850acR116-R146) for this one 🤦🏼 

Glad there's a linting rule to enable and will be sure to avoid this in the future 👍🏼